pyroute2/netlink/rtnl/req.py

```diff
@@ -119,6 +120,27 @@ def encap_header(self, header):
             if override_bos:
                 ret[-1]['bos'] = 1
             return {'attrs': [['MPLS_IPTUNNEL_DST', ret]]}
+        '''
+        Seg6 encap header transform. Format samples:
+
+            {'type': 'seg6',
+            'mode': 'encap',
+            'segs':'2000::5,2000::6'}
```

Comment:
how many segments there may be?

Reply:
AFAIK, the number can be very high. I think, looking at the elements of the header, it can be at most 256